src/sentry/uptime/models.py

```diff
@@ -154,6 +154,9 @@ def get_audit_log_data(self):
             "url": self.uptime_subscription.url,
             "interval_seconds": self.uptime_subscription.interval_seconds,
             "timeout": self.uptime_subscription.timeout_ms,
+            "method": self.uptime_subscription.method,
+            "headers": self.uptime_subscription.headers,
+            "body": self.uptime_subscription.body,
```

Comment:
unrelated?

Reply:
Yeah, something we missed when adding these new fields, just going to include it here